app/views/partials/chargeSummary/ChargeSummaryPaymentAllocation.scala.html

```diff
@@ -46,26 +47,28 @@
     </a>
 }
 
-@for(allocation <- paymentAllocations ; payment <- allocation.payments) {
+@for(allocation <- paymentAllocations ; payment <- allocation.allocations) {
     <tr>
         <td class="govuk-table__cell govuk-!-width-one-quarter">@{payment.dueDate.getOrElse(throw MissingFieldException("Payment Date")).toLongDateShort}</td>
         <td class="govuk-table__cell">
             @{
-                val paymentLot = payment.lot.getOrElse(throw MissingFieldException("Payment Lot"))
-                val paymentLotItem = payment.lotItem.getOrElse(throw MissingFieldException("Payment Lot Item"))
-                val matchingPayment = payments.findMatchingPaymentDocument(paymentLot, paymentLotItem)
+
```

Comment:
I think we need to rush merge this change as another team is working in the same page: https://jira.tools.tax.service.gov.uk/browse/MISUV-6912